### ISSUE TITLE
Wear OS tiles that need to be configured should take user to the appropriate location

### ISSUE THREAD

**Issue by dshokouhi** [home-assistant/android#4918]
**Is your feature request related to a problem? Please describe.**

Yes our current workflow for configuring tiles can be confusing to some where you either need to use the phone or watch.

**Describe the solution you'd like**

Add a button to unconfigured tile to take user to either watch or phone settings to set up.

**Describe alternatives you've considered, if any**

None while we can try to update the text to be more clear we only have so much space to work with.

**Additional context**

**Comment by Martreides** [comment]
I have started work on this enhancement and have a first working test. It now shows an "open settings" button on the thermostat tile which takes the users to the page to configure the tile based on the ID of the tile. Current version still contains comments an debug info, so need clean up. If the implementation is in line with expectations it should be quite easy to also implement it for the other tiles.

[Branch in my fork.](https://github.com/Martreides/android/tree/dev-tile-to-app)

**Comment by jpelgrom** [comment]
This was closed prematurely - there are other kinds of tiles that would also benefit from this.

**Comment by TimoPtr** [comment]
Good catch 👍 GitHub closed it because the description contains the resolved keyword...

**Comment by Martreides** [comment]
It has now been added to the camera tile as well: https://github.com/home-assistant/android/pull/5708
Will continue working on the other tiles.

**Comment by Martreides** [comment]
I was looking into the two last tiles that need this change, shortcut and template.

What is your opinion on how we should handle the template tile? It is configured in the phone app. Ideally it would open the app on the phone when the user clicks a button on the tile. However, that is quite a bit more complicated than what I have done so far to resolve this issue.
